cli/lib/cube.ts

```diff
@@ -41,7 +41,7 @@ export function injectRevision(this: Pick<Context, 'variables' | 'log'>) {
     return term
   }
 
-  return map(({ subject, predicate, object, graph }: Quad) => {
+  return map(async ({ subject, predicate, object, graph }: Quad) => {
```

Comment:
Won't have to be async if you also apply the change in the `.d.ts`
```suggestion
  return map(({ subject, predicate, object, graph }: Quad) => {
```